sui/wormhole/sources/state.move

```diff
@@ -87,14 +87,23 @@ module wormhole::state {
         chain_id: u64,
         governance_chain_id: u64,
         governance_contract: vector<u8>,
-        initial_guardian: vector<u8>,
+        initial_guardians: vector<vector<u8>>,
         _ctx: &mut TxContext
     ) {
         set_chain_id(&mut state, chain_id);
         set_governance_chain_id(&mut state, governance_chain_id);
         set_governance_contract(&mut state, governance_contract);
-        let initial_guardian = vector[structs::create_guardian(initial_guardian)];
-        store_guardian_set(&mut state, u32::from_u64(0), structs::create_guardian_set(u32::from_u64(0), initial_guardian));
+
+        let guardians = vector::empty<Guardian>();
+        let i = 0;
+        while (i < vector::length<vector<u8>>(&initial_guardians)){
+            vector::push_back<Guardian>(&mut guardians, structs::create_guardian(*vector::borrow<vector<u8>>(&initial_guardians, i)));
+            i = i + 1;
+        };
```

Comment:
I think it'd be good to keep code as consistent with the aptos version as we can
https://github.com/wormhole-foundation/wormhole/blob/5c0ed392d464d02de1d4bcb2d5ac4e4cedcb7a2c/aptos/wormhole/sources/wormhole.move#L70-L73

Reply:
changed, thanks